config/vufind/Blender.ini

```diff
@@ -0,0 +1,226 @@
+; displayed in the results list for each result. Use an empty string for no label.
+; Backend order defines the order the results are fetched from each backend.
+[Backends]
+Solr = "Local Index"
+Primo = "CDI"
+EDS = "EDS"
+
+[Blending]
+; Backends used for blending the first results for at most the initial 20 results:
+initialResults[] = Solr
+initialResults[] = Solr
+initialResults[] = Primo
+initialResults[] = EDS
+initialResults[] = Primo
+initialResults[] = EDS
+
+; The number of records to take from each backend when filling a result list in a
+; round-robin fashion:
+blockSize = 7
+
+; ---------- searches.ini settings ----------
+
+[General]
+default_handler = AllFields
+default_sort = "relevance,id asc"
+default_view = list
+default_limit = 20
+;limit_options        = 10,20,40,60,80,100
+case_sensitive_bools = true
+case_sensitive_ranges = true
+
+default_top_recommend[] = SpellingSuggestions
+;default_top_recommend[] = VisualFacets:Visual_Settings
+default_side_recommend[] = SideFacetsDeferred:Results:CheckboxFacets:Blender
+;default_noresults_recommend[] = SwitchTab
+default_noresults_recommend[] = SwitchType
+default_noresults_recommend[] = SwitchQuery:::fuzzy
+default_noresults_recommend[] = SpellingSuggestions
+default_noresults_recommend[] = RemoveFilters
+
+; Set this to true in order to highlight keywords from the search query when they
+; appear in fields displayed in search results.
+highlighting = true
+
+; Set this to restrict the list of fields that will be highlighted (the hl.fl
+; Solr parameter); default = '*' for all fields:
+highlighting_fields = "author,author2,author_additional,author_fuller,container_title,contents,culture,description,dewey-full,edition,era,genre,geographic,institution,isbn,issn,language,lccn,material,measurements,publication_place_txt_mv,publisher,series,series2,title,title_alt,title_full,title_new,title_old,topic"
+snippets = true
+retain_filters_by_default = true
+;default_filters[] = "format:Book"
+;default_filters[] = "institution:MyInstitution"
+;default_filters[] = "(format:Book AND institution:MyInstitution)"
+
+;display_versions = true
+;versions_filter = "same_source"
+
+[Records]
+; Boolean value indicating if deduplication is enabled. Defaults to false.
+;deduplication = true
+; Priority order (descending) for record sources (record ID prefixes separated
+; from the actual record by period, e.g. testsrc.12345)
+;sources = alli,testsrc
+
+; This section controls the search handler options displayed in blended mode.
+[Basic_Searches]
+AllFields = "All Fields"
+Title = Title
+Author = Author
+Subject = Subject
+
+[Sorting]
+relevance = sort_relevance
+year = sort_year
+year asc = "sort_year asc"
+
+; ---------- facets.ini settings ----------
+
+; The order of display is as shown below
+; The name of the index field is on the left
+; The display name of the field is on the right
+[Results]
+institution        = Institution
+building           = Organisation
```

Comment:
I don't think "Organisation" is in the language files (and this is a British spelling); we should probably try to align the example config with existing options where possible (or add translations).

Reply:
Fixed.